app/policies/collection_policy.rb

```diff
@@ -16,6 +16,11 @@ def update?
     administrator? || manages_collection?(record)
   end
 
+  sig { returns(T::Boolean) }
+  def edit_first_draft?
+    update? && record.collection_versions.first.first_draft?
+  end
```

Comment:
we are only allowed to edit a first draft of a collection if its actually a first draft -- this ensures we don't end up on that page otherwise